target/zephyr/Kconfig

```diff
@@ -149,6 +149,82 @@ if MENDER_MCU_CLIENT
                 the bootloader configuration, e.g. `sysbuild.conf` if you're using sysbuild.
     endmenu
 
+    menu "Mender Artifact configuration"
+
+        config MENDER_ARTIFACT_GENERATE
+            bool "Generate a Mender Artifact from the built Zephy kernel image"
+            default y
+            help
+                Auto-generates a Mender Artiafact from the Zephyr kernel. It requires the mender-artifact CLI tool
+                installed in the workstation.
```

Comment:
This could be done with `menuconfig` instead of `menu` + `config`. Not a biggie, but I think it's nicer.

Reply:
What are you referring to? I don't get it.

Reply:
`menuconfig` is what creates this `[*] A submenu you can disable as a whole`. I think that's the way to add a group of things that all depend on something being enabled/disabled. See https://www.kernel.org/doc/html/latest/kbuild/kconfig-language.html for details.

Reply:
I got it now. Thanks.